tests/unit/Infrastructure/Manage/Client/DeleteEntityClientTest.php

```diff
@@ -117,4 +118,36 @@ public function provideInvalidProtocols()
         yield ['oauth_ccc'];
         yield ['i-dont-exist'];
     }
+
+    /**
+     * @dataProvider provideExpectedProtocolMappings
+     */
+    public function test_it_maps_enitty_types_correctly_with_excisting_protocols($inputProtocol, $expectedProtocol)
+    {
+        $this->mockHandler = new MockHandler();
+        $client = m::mock(HttpClientInterface::class);
+        $this->generator = m::mock(GeneratorInterface::class);
+        $this->logger = m::mock(LoggerInterface::class);
+        $this->client = new DeleteManageEntityClient(
+            $client,
+            $this->logger
+        );
```

Comment:
I copy pasted this from the setUp method during our peer programming session. Now that I'm looking at this code more closely I see you/we can leave out several of the field declarations (generator, logger, mockHandler. They can all be removed from this scope.

Reply:
Cleaned up the test case